controller.go

```diff
@@ -841,6 +845,7 @@ addToStore:
 	if !c.isDistributedControl() {
 		c.Lock()
 		arrangeIngressFilterRule()
+		arrangeUserFilterRule()
 		c.Unlock()
 	}
 
```

Comment:
@wenjianhn `arrangeIngressFilterRule()` is done inside the `!c.isDistributedControl()` because it applies only in the swarm mode. But the user chain is needed in non-swarm mode as well. Can you add this after the `if !c.isDistributedControl() ` block
```
 		c.Lock()
 		arrangeUserFilterRule()
  		c.Unlock()
```